docs/man-pages/htcondor.rst

```diff
@@ -319,6 +325,35 @@ distribute to jobs which request them.
     directory (:macro:`SEC_CREDENTIAL_DIRECTORY_OAUTH`); in most cases,
     this means as ``root``.
 
+System Verbs
+------------
+
+The *system* noun represents the local HTCondor installation of the
+current host.
```

Comment:
If you're going to include this line for "system", it should be there for "access point" and "central manager", as well, since those two are probably less clear.

Consider something like the following:

> The system, access-point, and central-manager nouns refer to different parts of an HTCondor pool.
...
> The system noun refers to all the HTCondor daemons on the local system.
...
> The access-point noun should be plural, and refers to every access point known to the pool.  This list will be looked up in the pool's collector(s).
...
> The central-manger noun should be plural, and refers to every collector to which the local host reports, as set in :macro:`COLLECTOR_HOST`.  (Athough a HTCondor pool is normally defined by its collector, a pool may have more than one collector to ensure high availability [link]; this configuration is also used for startd flocking [link].)
...
> In all three cases, the **diagnose** verb attempts to determine the health of each of the daemon(s) in question.